languages/r36/template/algorithmia.conf

```diff
@@ -0,0 +1,5 @@
+{
+    "username": "__USER__",
+    "algoname": "__ALGO__",
+    "langauge": "r36"
```

Comment:
misspelling of "language" says "langauge" - prob someone already caught that tho